pkg/runtime/executor.go

```diff
@@ -139,3 +168,68 @@ func (e *Executor) Execute() (results Output, err error) {
 	// successful
 	return results, nil
 }
+
+// read the config file and update scan and skip rules
+func (e *Executor) initScanAndSkipRules() error {
+	if e.configFile != "" {
+		configData, err := config.LoadConfig(e.configFile)
```

Comment:
It should not be the responsibility of this package to `LoadConfig`, it should happen in `config` package. There should be handlers in the `config` package to return what's required by this package.

Reply:
done as part of config reader refactor